server/src/main/java/com/coincoinche/websockets/controllers/GameController.java

```diff
@@ -5,16 +5,10 @@
 import com.coincoinche.engine.Move;
 import com.coincoinche.engine.MoveBidding;
 import com.coincoinche.engine.MovePlaying;
+import com.coincoinche.engine.game.GameResult;
 import com.coincoinche.engine.teams.Player;
-import com.coincoinche.events.BiddingTurnStartedEvent;
-import com.coincoinche.events.CardPlayedEvent;
-import com.coincoinche.events.Event;
-import com.coincoinche.events.EventType;
-import com.coincoinche.events.PlayerBadeEvent;
-import com.coincoinche.events.PlayingTurnStartedEvent;
-import com.coincoinche.events.RoundPhaseStartedEvent;
-import com.coincoinche.events.RoundStartedEvent;
-import com.coincoinche.events.TurnStartedEvent;
+import com.coincoinche.engine.teams.Team;
+import com.coincoinche.events.*;
```

Comment:
We need to replace this by a regular import.

Reply:
Webstrom does that automatically :/

Reply:
Done, I changed my settings